### DISCUSSION TITLE
Examples of octave

### DISCUSSION THREAD

**Discussion by cmat21** [ra3xdh/qucs_s#628] [Q&A] [answered]
Hello, could you share examples of using octave or some related documentation (or video)?
thanks in advance

**Comment by ra3xdh** [comment, marked as answer]
The stuff in `share/octave` subdirectory was not updated long time ago. I don't know if it is operational. Furthermore I never used this stuff and Octave integration from the main window. I recommend to export the plot data using right click on trace and then `Export to CSV` menu. Then you can import CSV in Octave and do whatever you want.
